doc/api/api_changes/2015-04-18-drawingarea.rst

```diff
@@ -0,0 +1,8 @@
+'OffsetBox.DrawingArea' no longer accepts the 'clip' keyword argument
+`````````````````````````````````````````````````````````````````````
+
+The call signature was `OffsetBox.DrawingArea(..., clip=True)` but nothing
+was done with the `clip` argument. The object did not do any clipping
+regardless of that parameter. Now the object can and does clip the child `Artists` if they are set to be clipped.
+
+You can turn off the clipping using :method:`Artist.set_clip_on(False)`
```

Comment:
By changed, I mean what artist do you call `set_clip_on(False)` on.  It is not clear from this text that it should be on the children, not on the DrawingArea.